### PR TITLE
Navigation change for Indian Sportswoman of the Year

### LINKED ISSUE #5326: Navigation change for Indian Sportswoman of the Year
**Is your feature request related to a problem? Please describe.**
Add the following item in the **last position** of the nav bar of each India sites:

> **BBC Gujarati**
> item: &iswofty
> title: #ISWOTY 
> link; /gujarati/resources/idt-0370808c-2dea-4f55-b69f-309a2881ef25
> 
>  
> **BBC HIndi**
> item: &iswofty
> title: #ISWOTY 
> link; /hindi/resources/idt-a6da5349-3698-4f42-9e5b-35513c8c0537
> 
>  
> **BBC Marathi**
> item: &iswofty
> title: #ISWOTY 
> link; /marathi/resources/idt-43ff5603-f85f-47c1-972f-097e67eeb035
> 
> 
> **BBC Punjabi**
> item: &iswofty
> title: #ISWOTY 
> link; /gujarati/resources/idt-0370808c-2dea-4f55-b69f-309a2881ef25
> 
>  
> **BBC Tamil**
> item: &iswofty
> title: #ISWOTY 
> link; /tamil/resources/idt-ce219e77-9003-42b8-b276-1214e232fdc0
> 
>  
> **BBC Telugu**
> item: &iswofty
> title: #ISWOTY 
> link; /telugu/resources/idt-47054a9b-b97c-4190-bed0-0416210e698b

**Describe the solution you'd like**
Update services config files with relevant navigation items.

**Describe alternatives you've considered**
N/A

**Testing notes**
- [ ] This feature is expected to need manual testing.

**Additional context**
Relevant JIRA ticket: WSFRONTPGS-272

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #5326

**Overall change:** 
Added links for Sportswoman of the Year to navigation for Gujarati, Hindi, Marathi, Punjabi, Tamil and Telegu.

**Code changes:**
- Updated services config files with relevant navigation items.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/punjabi.js [hunk 1]
```diff
@@ -169,6 +169,10 @@ export const service = {
         title: 'ਕੌਮਾਂਤਰੀ',
         url: '/punjabi/international',
       },
+      {
+        title: '#ISWOTY',
+        url: '/gujarati/resources/idt-0370808c-2dea-4f55-b69f-309a2881ef25',
```
Comment (j-pendlebury) [inline src/app/lib/config/services/punjabi.js]: weird that this is a link to Gujarati. I can see this link on the old front page: `/punjabi/resources/idt-681afed0-03f5-4d5f-a640-3011a722d1c8`. Could you double check with Jon?

Comment (OlgaLyubin) [reply src/app/lib/config/services/punjabi.js]: Good catch 👍Double checked with Jon and updated the link

**Review by j-pendlebury — APPROVED** [review]

**Review by ryanmccombe — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Comment by staylos92** [comment]
This ticket does not need testing by a tester. Any testing would just be that the nav items were correct but that should have been checked as part of the development and code review phases anyway. 👍

**Pull request merged**